.github/workflows/crates_upload.yml

```diff
@@ -0,0 +1,43 @@
+name: crates_upload
+
+on:
+  release:
+    types: [published]
+  push:
+    branches: [main]
+
+permissions:
+  contents: read
+
+jobs:
+  upload_release:
+    name: Upload release to crates.io
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+
+      - name: Publish libcst_derive to crates.io
+        uses: actions-rs/cargo@v1
+        with:
+          command: publish
+          args: >
+            --manifest-path ./native/libcst_derive/Cargo.toml
+            --allow-dirty
```

Comment:
why `--allow-dirty`?

Reply:
I was testing locally with [nektos/act](https://github.com/nektos/act) and was getting an issue where `cargo publish` wouldn't allow files that weren't committed, even though the files were actually committed. Not sure if this would be the case on GitHub